#! /bin/sh /usr/share/dpatch/dpatch-run
## 25manpage_note_dropped_F.dpatch by  <hesso@pool.math.tu-berlin.de>
##
## DP: Replace the manpage paragraph about the '-F' commandline option
## DP: with a notice that is has been dropped.

@DPATCH@
--- nvi-1.81.6+debian-1.orig/docs/vi.man/vi.1	2008-06-13 00:52:45.000000000 +0200
+++ nvi-1.81.6+debian-1/docs/vi.man/vi.1	2008-06-13 00:55:40.000000000 +0200
@@ -17,7 +17,7 @@
 .SH SYNOPSIS
 .B ex
 [\c
-.B \-eFRrSsv\c
+.B \-eRrSsv\c
 ] [\c
 .BI \-c " cmd"\c
 ] [\c
@@ -28,7 +28,7 @@
 .br
 .B vi
 [\c
-.B \-eFlRrSv\c
+.B \-elRrSv\c
 ] [\c
 .BI \-c " cmd"\c
 ] [\c
@@ -39,7 +39,7 @@
 .br
 .B view
 [\c
-.B \-eFRrSv\c
+.B \-eRrSv\c
 ] [\c
 .BI \-c " cmd"\c
 ] [\c
@@ -113,11 +113,6 @@
 Start editing in ex mode, as if the command name were
 .IR \&ex .
 .TP
-.B \-F
-Don't copy the entire file when first starting to edit.
-(The default is to make a copy in case someone else modifies
-the file during your edit session.)
-.TP
 .B \-l
 Start editing with the lisp and showmatch options set.
 .TP
@@ -169,6 +164,9 @@
 or
 .IR view .
 .PP
+Note that the \fB-F\fP option (which prevented \fIex/vi\fP from making
+a full backup of the target file) has been removed and is no longer available.
+.PP
 Command input for
 .I ex/vi
 is read from the standard input.
